docker/Real_device

```diff
@@ -69,9 +69,36 @@ RUN apt-get -qqy update && apt-get -qqy install --no-install-recommends \
     net-tools \
     ffmpeg \
     jq \
+    curl \
+    libavcodec-dev \
+    libavdevice-dev \
+    libavfilter-dev \
+    libavformat-dev \
+    libavresample-dev \
+    libavutil-dev \
+    libpostproc-dev \
+    libswresample-dev \
+    libswscale-dev \
+    gcc \
+    git \
+    make \
+    meson \
+    musl-dev \
+    pkgconf \
+    libsdl2-dev \
  && apt clean all \
  && rm -rf /var/lib/apt/lists/*
 
+ARG SCRCPY_VER=1.10
+ARG SERVER_HASH="cbeb1a4e046f1392c1dc73c3ccffd7f86dec4636b505556ea20929687a119390"
+
+RUN PATH=$PATH:/usr/lib/jvm/java-1.8-openjdk/bin
+RUN git clone https://github.com/Genymobile/scrcpy.git
```

Comment:
I think we dont need to clone the repo or I miss something?

Reply:
Well, another option would be to get a prebuild scrcpy binary from somewhere.

Reply:
In my opinion it is better to use the stable version that scrcpy released which you did in the next line. Besides clone the repo just make the size bigger.

Reply:
There is no Linux binary release. Only for the server jar. But fetching a released code zip and building from that instead might be a good idea stability wise.

Reply:
We are now building from released zip, installing and removing sources afterwards.